src/core/hashTable.cc

```diff
@@ -246,21 +246,20 @@ T_mv HashTable_O::maphash(T_sp function_desig) {
 CL_LAMBDA(function-desig hash-table);
 CL_DECLARE();
 CL_DOCSTRING("see CLHS");
-CL_DEFUN T_mv cl__maphash(T_sp function_desig, T_sp thash_table) {
+CL_DEFUN T_mv cl__maphash(T_sp function_desig, HashTableBase_sp hash_table) {
```

Comment:
Could you change this to return a T_sp?   T_mv is for Values(...) but T_sp is more correct as you point out.

Reply:
I only correct half of the problem as you spotted will do as you said and look for other occurances